icons/svg/file_type_esbuild.svg

```diff
@@ -0,0 +1,11 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
```

Comment:
1. Icons with heavy background should be of 14x14 pixels while leaving 1px margin around each side. Otherwise they appear too big in ST.
2. Please ungroup the svg after changing size, so only `<path>` elements are present. Also `style` tags may be problematic. Use `fill=#rrggbb` instead.